rootfs/rootfs/etc/rc.d/automount

```diff
@@ -34,6 +34,37 @@ if [ ! -n "$BOOT2DOCKER_DATA" ]; then
             mkfs.ext4 -L $LABEL $BOOT2DOCKER_DATA
             swapon "${UNPARTITIONED_HD}2"
         fi
+
+        DISK_VENDOR=$(cat /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/vendor /sys/class/block/$(basename $UNPARTITIONED_HD /dev/)/device/model | tr -d "\n")
+        # Test if disk is "VMware, VMware Virtual S" and empty.
+        if [ "$DISK_VENDOR" = "VMware, VMware Virtual S" ] || [ "$DISK_VENDOR" = "VMware  Virtual disk    " ]; then
```

Comment:
Ouch.  I didn't realize `[[`'s pattern-matching wasn't POSIX.  That hurts.

Here's a hack that _does_ work, though!
```console
$ docker run -it --rm busybox
/ # vmware='some VMware something'
/ # [ "${vmware%%*VMware*}" != "$vmware" ] && echo yes || echo no
yes
```